extensions/azurecore/package.json

```diff
@@ -19,6 +19,13 @@
     "url": "https://github.com/Microsoft/azuredatastudio.git"
   },
   "contributes": {
+    "resourceViewResources": [
```

Comment:
did we want this to only show when the user has the azurearc extension installed in the Sep release?  In that case, should we add this contribution to the azure arc extension?

Reply:
This is how resources get contributed. If we want to view to only show when an extension is installed I can do that separately.